# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libdispatch/hotfixes.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- swift-corelibs-libdispatch-swift-4.2.2-RELEASE/src/shims/generic_unix_stubs.c.vanilla	2020-11-14 17:06:38.814641625 +0100
+++ swift-corelibs-libdispatch-swift-4.2.2-RELEASE/src/shims/generic_unix_stubs.c	2020-11-14 17:06:53.106641825 +0100
@@ -53,5 +53,5 @@
 unsigned short dispatch_callout__entry_semaphore;
 unsigned short dispatch_callout__return_semaphore;
 unsigned short dispatch_queue__push_semaphore;
-void (*_dispatch_block_special_invoke)(void*);
-struct dispatch_queue_attr_s _dispatch_queue_attr_concurrent;
+//void (*_dispatch_block_special_invoke)(void*);
+//struct dispatch_queue_attr_s _dispatch_queue_attr_concurrent;
--- swift-corelibs-libdispatch-swift-4.2.2-RELEASE/src/queue.c.vanilla	2020-11-14 17:03:17.226638801 +0100
+++ swift-corelibs-libdispatch-swift-4.2.2-RELEASE/src/queue.c	2020-11-14 17:03:33.742639033 +0100
@@ -920,12 +920,14 @@
 
 #ifndef __ANDROID__
 #ifdef SYS_gettid
+#if 0
 DISPATCH_ALWAYS_INLINE
 static inline pid_t
 gettid(void)
 {
 	return (pid_t)syscall(SYS_gettid);
 }
+#endif
 #elif defined(__FreeBSD__)
 DISPATCH_ALWAYS_INLINE
 static inline pid_t
